QuoridorEngine/src/ui/gui/Board.cs

```diff
@@ -0,0 +1,182 @@
+﻿using QuoridorEngine.src.ui.gui.board;
+using QuoridorEngine.UI;
+
+namespace QuoridorEngine.src.ui.gui
+{
+#if !CONSOLE 
+    public class Board : Panel
```

Comment:
Maybe change class name to something like BoardUI to avoid confusion with class QuoridorBoard